### PR TITLE
Header, new header, footer and side menu traffic for the new support US page

### PR DESCRIPTION
**Pull request by svillafe**
## What does this change?
Change the traffic for the US to the new support site

## What is the value of this and can you measure success?
N/A
## Does this affect other platforms - Amp, Apps, etc?
N/A

## Screenshots
N/A

## Tested in CODE?
Not yet

### CODE REVIEW

File: common/app/navigation/helpers/UrlHelpers.scala [hunk 1]
```diff
@@ -118,37 +121,31 @@ object UrlHelpers {
...
-      getReaderRevenueUrl(Contribute, position)
-    }
+    countryUrlLogic(editionId, position, Contribute)
   }
 
   def getSupportOrSubscriptionUrl(position: Position)(implicit request: RequestHeader): String = {
     val editionId = Edition(request).id.toLowerCase()
-    if (editionId == "uk" && SupportFrontendActive.isSwitchedOn) {
+    if (editionId == "uk" && UkSupportFrontendActive.isSwitchedOn) {
       getReaderRevenueUrl(Support, position)
```
Comment (JustinPinner) [inline common/app/navigation/helpers/UrlHelpers.scala]: What do we expect to happen for the US here?

Comment (svillafe) [reply common/app/navigation/helpers/UrlHelpers.scala]: Nothing, the US's subscription link keeps pointing to the subs site.

**Comment by JustinPinner** [comment]
Looks good to me 👍

**Review by joelochlann — APPROVED** [review]
Looks good!

**Review by SiAdcock — APPROVED** [review]
👍

**Comment by svillafe** [comment]
It seems it is working in PROD (the switch is in the switchboard)

**Pull request merged**